ppl/src/test/java/org/opensearch/sql/ppl/PPLServiceTest.java

```diff
@@ -161,7 +161,7 @@ public void onFailure(Exception e) {
   public void testPrometheusQuery() {
     doAnswer(invocation -> {
       ResponseListener<QueryResponse> listener = invocation.getArgument(1);
-      listener.onResponse(new QueryResponse(schema, Collections.emptyList()));
+      listener.onResponse(new QueryResponse(schema, Collections.emptyList(), ""));
```

Comment:
You can define new constructor in `QueryResponse` class which accepts 2 arguments to avoid such changes.

Reply:
This will be addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/217 as it is not part of legacy fallback.